pkg/handlers/v1/elb.go

```diff
@@ -28,68 +35,104 @@ func extractELBNetworkInfo(config *elbConfiguration) Change {
 
 type elbTransformer struct{}
 
-func (t elbTransformer) Create(event awsConfigEvent) (Output, error) {
+// Classic ELBs have createdTime in the form of milliseconds, which is int64 in Go,
```

Comment:
I appreciate these comments on why we're doing this a lot. Pieces of domain knowledge like this tend to get forgotten months out from when we first write code and become a pain to relearn when we have to touch the codebase later